[Bug] piqa or lambada dataset is not available

### Prerequisite

- [x] I have searched [Issues](https://github.com/open-compass/opencompass/issues/) and [Discussions](https://github.com/open-compass/opencompass/discussions) but cannot get the expected help.
- [x] The bug has not been fixed in the [latest version](https://github.com/open-compass/opencompass).

### Type

I'm evaluating with the officially supported tasks/models/datasets.

### Environment

I use python run.py --datasets piqa_gen --hf-type base --hf-path /root/autodl-tmp/deepseek-Qwen-1.5B --debug
but get error
other dataset like aime or math or HellaSwag or WinoGrande is avaliable, piqa and lambada dataset get error

### Reproduces the problem - code/configuration sample

python run.py --datasets piqa_gen --hf-type base --hf-path /root/autodl-tmp/deepseek-Qwen-1.5B --debug

### Reproduces the problem - command or script

python run.py --datasets piqa_gen --hf-type base --hf-path /root/autodl-tmp/deepseek-Qwen-1.5B --debug

### Reproduces the problem - error message

(quarot) root@autodl-container-fee8478cb9-083f90fa:~/autodl-tmp/opencompass# python run.py --datasets piqa_gen --hf-type base --hf-path /root/autodl-tmp/deepseek-Qwen-1.5B --debug --batch-size 50

07/08 18:19:02 - OpenCompass - INFO - Loading piqa_gen: /root/autodl-tmp/opencompass/opencompass/configs/./datasets/piqa/piqa_gen.py
07/08 18:19:02 - OpenCompass - INFO - Loading example: /root/autodl-tmp/opencompass/opencompass/configs/./summarizers/example.py
07/08 18:19:02 - OpenCompass - INFO - Current exp folder: outputs/default/20250708_181902
07/08 18:19:02 - OpenCompass - WARNING - SlurmRunner is not used, so the partition argument is ignored.
07/08 18:19:02 - OpenCompass - INFO - ./data/piqa does not exist!Start Download data automatically!If you have downloaded the data before,You can specific `COMPASS_DATA_CACHE` to avoid downloading~
Traceback (most recent call last):
  File "/root/autodl-tmp/opencompass/run.py", line 5, in <module>
    main()
  File "/root/autodl-tmp/opencompass/opencompass/cli/main.py", line 330, in main
    tasks = partitioner(cfg)
  File "/root/autodl-tmp/opencompass/opencompass/partitioners/base.py", line 83, in __call__
    tasks = self.partition(**model_and_dataset_args,
  File "/root/autodl-tmp/opencompass/opencompass/partitioners/num_worker.py", line 69, in partition
    dataset_size = self.get_size(dataset)
  File "/root/autodl-tmp/opencompass/opencompass/partitioners/num_worker.py", line 141, in get_size
    dataset = build_dataset_from_cfg(dataset)
  File "/root/autodl-tmp/opencompass/opencompass/utils/build.py", line 12, in build_dataset_from_cfg
    return LOAD_DATASET.build(dataset_cfg)
  File "/root/miniconda3/envs/quarot/lib/python3.10/site-packages/mmengine/registry/registry.py", line 570, in build
    return self.build_func(cfg, *args, **kwargs, registry=self)
  File "/root/miniconda3/envs/quarot/lib/python3.10/site-packages/mmengine/registry/build_functions.py", line 121, in build_from_cfg
    obj = obj_cls(**args)  # type: ignore
  File "/root/autodl-tmp/opencompass/opencompass/datasets/base.py", line 17, in __init__
    dataset = self.load(**kwargs)
  File "/root/autodl-tmp/opencompass/opencompass/datasets/piqa.py", line 83, in load
    path = get_data_path(path)
  File "/root/autodl-tmp/opencompass/opencompass/utils/datasets.py", line 52, in get_data_path
    return download_dataset(local_path, cache_dir)
  File "/root/autodl-tmp/opencompass/opencompass/utils/datasets.py", line 90, in download_dataset
    assert dataset_name, f'No valid url for {data_path}!\n' + \
AssertionError: No valid url for ./data/piqa!
Please make sure  `./data/piqa` is correct

### Other information

_No response_

--- a/opencompass/utils/datasets_info.py
+++ b/opencompass/utils/datasets_info.py
@@ -680,6 +680,10 @@ DATASETS_URL = {
         "url": "http://opencompass.oss-cn-shanghai.aliyuncs.com/datasets/data/lcb_pro.zip",
         "md5": "ff4572e292fe27cabce3ab3b407361bd",
     },
+    "/lambada/": {
+        "url": "http://opencompass.oss-cn-shanghai.aliyuncs.com/datasets/data/lambada.zip",
+        "md5": "d001d2125743c8aa4d770d4dd1718f29",
+    },
     "/mol-instructions": {
         "url": "http://opencompass.oss-cn-shanghai.aliyuncs.com/datasets/data/mol-instructions.zip",
         "md5": "ea3db4a199222835036235b18306065a",
@@ -752,6 +756,10 @@ DATASETS_URL = {
         "url":
         "http://opencompass.oss-cn-shanghai.aliyuncs.com/datasets/data/gpqa.zip",
         "md5": "2e9657959030a765916f1f2aca29140d",
+    },
+    "/piqa": {
+        "url": "http://opencompass.oss-cn-shanghai.aliyuncs.com/datasets/data/piqa.zip",
+        "md5": "39374924aec45f2f7efdffb1977ce2ba",
     },
     "/CHARM/": {
         "url":

